cypress/support/commands.js

```diff
@@ -32,18 +14,27 @@ Cypress.Commands.add('login', (email, pw, userName) => {
 }); 
 
 Cypress.Commands.add('logout', ()=> {
-    cy.get('.DweEFaF5owOe02').click();
+    cy.get('[class="OUdAuicP657Tka"]').click();
     cy.get('[data-testid="account-menu-logout"]').click();
 });
 
-Cypress.Commands.add('boardDelete', (boardName) => {
-    cy.get('[class="board-tile-details-name"]').contains(boardName).click();
-    cy.get('[class="frrHNIWnTojsww GDunJzzgFqQY_3 bxgKMAm3lq5BpA HAVwIqCeMHpVKh SEj5vUdI3VvxDc"]').click({force: true});
-    cy.get('[class="board-menu-navigation-item-link js-open-more"]').click();
-    cy.get('[class="board-menu-navigation-item-link js-close-board"]').click({force: true});
-    cy.get('[class="js-confirm full nch-button nch-button--danger"]').click(); 
-    cy.get('[data-testid="close-board-delete-board-button"]').click();
-    cy.get('[data-testid="close-board-delete-board-confirm-button"]').click();
+// HANDLING WORKSPACES AND BOARD WITH UI.
+Cypress.Commands.add('createWorkSpace', (workSpaceName)=> {
+    // I did triple-click for a bug in this button.
+    cy.get('[data-testid="header-create-menu-button"]').dblclick().click();
```

Comment:
Is it a bug, or maybe another element is being clicked, you can try the option: `{force: true}`